plugins/verifier/cosign/cosign.go

```diff
@@ -146,10 +146,9 @@ func VerifyReference(args *skel.CmdArgs, subjectReference common.Reference, refe
 		// The verification will return an error if the signature is not valid.
 		// Currently the bundle verification result is ignored
 		_, err = cosign.VerifyImageSignature(ctx, sig, subjectDescHash, cosignOpts)
-		if err != nil {
```

Comment:
Recommend adding a log here so that we know what are the possible errors that cosign is generating.

Reply:
@suganyas and I discussed offline that adding error log here will not surface to the user in this plugin scenario. Because the cosign plugin runs as a separate process, Ratify will parse the stdout from the plugin and extracts only the verification result. It ignores all other statements currently so the error log would never surface. We probably need to revisit this...